buildSrc/src/main/kotlin/spotless-java-extension.gradle.kts

```diff
@@ -58,12 +79,34 @@ class MultilineFormatter : Serializable {
             val minIndent =
                 multilineStringLines
                     .filter { it.isNotBlank() }
-                    .map { l -> l.indexOfFirst { ch -> !ch.isWhitespace() }.takeIf { it >= 0 } ?: line.length }
-                    .minOrNull() ?: 0
+                    .minOfOrNull { l -> l.indexOfFirst { ch -> !ch.isWhitespace() }.takeIf { it >= 0 } ?: line.length }
+                    ?: 0
             multilineStringLines.forEach { blockLine ->
                 result.append(" ".repeat(baseIndent)).append(blockLine.drop(minIndent)).append("\n")
             }
         }
         return result.toString()
     }
+}
+
+class MultilineWithParameterFormatter : Serializable {
+    fun format(content: String): String {
+        val lines = content.lines()
+        val result = StringBuilder()
+        var i = 0
+        while (i < lines.size) {
+            val line = lines[i]
+            if (line.trimEnd().endsWith("\"\"\"") && i + 1 < lines.size && lines[i + 1].trimStart().startsWith(".formatted(")) {
+                result.append(line.trimEnd() + lines[i + 1].trimStart())
+                i++
+            } else {
+                result.append(line)
+            }
+            i++
+            if (i < lines.size) {
+                result.append("\n")
+            }
+        }
+        return result.toString()
+    }
```

Comment:
we might well merge the above logic into existing formatter which mainly aims for indentation, but that would make the code logic difficult to understand. One step one concern makes for much maintainable code.